Casks/p/processspy.rb

```diff
@@ -0,0 +1,23 @@
+cask "processspy" do
+  version "1.6.0"
+  sha256 "a7426398bcd5e2c5b7d72b3288f5d74c63e12179e2eceb14a6a90d935ff3f8bd"
+
+  url "https://process-spy.app/archive/ProcessSpy_#{version}.dmg"
+  name "ProcessSpy"
+  desc "Monitor processes with command-line arguments"
+  homepage "https://process-spy.app/"
+
+  livecheck do
+    url "https://process-spy.app/archive/appcast.xml"
+    strategy :sparkle, &:short_version
+  end
+
```

Comment:
```suggestion

  auto_updates true
```
<img width="549" alt="Screenshot 2024-07-24 at 15 22 43" src="https://github.com/user-attachments/assets/7d4bcc94-a621-4158-a4ad-b882768dc9f0">